recipes/ifitpy/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
You do not need to add this copy of the license since there is already a license file included in the tarball you're using for the source distribution. Please remove this file and change the license file in the `meta.yaml` to `LICENSE.txt`